data/layout/telescope_positions-North-4MST.ecsv

```diff
@@ -52,7 +52,7 @@
 #       value: 3.0
 # schema: astropy-2.0
 telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
```

Comment:
Is the altitude correct? center_alt is at 2150m, and this is probably as well the altitude we put into corsika. So all telescopes should at or above center_alt.